compiler/damlc/daml-prim-src/GHC/Types.daml

```diff
@@ -169,6 +169,18 @@ data Nat
 
 -- | A type for fixed-point decimal numbers, with the scale
 -- being passed as part of the type.
+--
+-- `Numeric n` represents a fixed-point decimal number with a
+-- fixed precision of 38 (i.e. 38 digits not including a leading zero)
+-- and a precision of `n`, i.e., `n` digits after the decimal point.
```

Comment:
```suggestion
-- and `n` decimals, i.e., `n` digits after the decimal point.
```
Otherwise *precision* is overloaded in this paragraph.

Reply:
Wouldn't the typical terminology be "a scale of `n`, i.e. ..."?

Reply:
oops yeah, that should have been scale. fixed.